index.html

```diff
@@ -1,13 +1,49 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <link rel="icon" type="image/svg+xml" href="/logo.webp" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Hackafor 2024</title>
-  </head>
-  <body>
-    <div id="root"></div>
-    <script type="module" src="/src/main.tsx"></script>
-  </body>
+
+<head>
+  <meta charset="UTF-8" />
+  <link rel="icon" type="image/svg+xml" href="/logo.webp" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Hackafor 2024</title>
+</head>
+
+<body>
+  <div id="root"></div>
+  <script type="module" src="/src/main.tsx"></script>
+  <noscript>
+    <style>
+      body {
+        background-color: #0e7490aa;
+        min-height: 100svh;
+        display: grid;
+        place-content: center;
+      }
+
+      .main {
+        margin-inline: 1rem;
+        display: flex;
+        align-items: center;
+        flex-direction: column;
+      }
+
+      .paragraph {
+        font-size: 2rem;
+        color: #292524;
+        text-align: center;
+      }
+
+      img {
+        max-width: 100%;
+      }
+    </style>
+    <main class="main">
+      <p class="paragraph">Lo siento, necesitas activar JavaScript para ver la
+        página 🥺 </p>
+      <img src="/images/hamtafor.webp">
```

Comment:
```suggestion
      <img src="/images/hamtafor.webp" alt="Afordin Hamster version">
```

Agregar el alt :3

Reply:
Good feedback! I'd also delete the 'atte afor digital' and I would only put the name in the alt image